src/core/layertree/qgslayertreemodellegendnode.h

```diff
@@ -361,6 +389,12 @@ class CORE_EXPORT QgsSymbolLegendNode : public QgsLayerTreeModelLegendNode
     // ident the symbol icon to make it look like a tree structure
     static const int INDENT_SIZE = 20;
 
+    /**
+     * Create an expressionContextScope containing symbol related variables
+     * \since QGIS 3.10
+     */
+    QgsExpressionContextScope *createSymbolScope() const;
```

Comment:
```suggestion
    QgsExpressionContextScope *createSymbolScope() const SIP_FACTORY;
```

And add a note to the doc string that ownership is transferred to the caller

Reply:
done